plugins/bazaar-backend/src/service/router.ts

```diff
@@ -70,36 +69,48 @@ export async function createRouter(
     }
   });
 
-  router.get('/metadata/:ref', async (request, response) => {
+  router.get('/projects/id/:id', async (request, response) => {
```

Comment:
would it make sense to just have a `/projects/:idOrRef` and detect whether it's all digits or not? and then pick internally which path to take